cmd/npmExecuteScripts.go

```diff
@@ -47,18 +48,32 @@ func runNpmExecuteScripts(npmExecutor npm.Executor, config *npmExecuteScriptsOpt
 		return err
 	}
 
-	log.Entry().Infof("creating build settings information...")
+	log.Entry().Debugf("creating build settings information...")
```

Comment:
as above, possible candidate for a re-use function

Reply:
Fix